src/remoteSaver.ts

```diff
@@ -111,6 +112,37 @@ export class WebAppRemoteSaver extends RemoteSaver<WebApp> {
     }
 }
 
+export class WikiArticleRemoteSaver extends RemoteSaver<WikiArticle> {
+    protected async printSaveSuccessMsg() {
+        window.showInformationMessage(`The wiki article has been saved in DSS!`);
+    }
+
+    protected async saveInDss(WikiArticle: WikiArticle): Promise<void> {
+        await saveWikiArticle(WikiArticle);
+    }
+
+    protected async cancelSave(remoteWikiArticle: WikiArticle): Promise<void> {
+        await this.fsManager.saveInFS(FileDetails.fromWikiArticle(remoteWikiArticle));
+    }
+
+    protected getConflictMessage(conflictingElement: WikiArticle): string {
```

Comment:
Great idea!
(I decided to keep `getConflictMessage()` in the children class to avoid having a big conditional structure in the parent method, with potential "dirty" casts to access `versionTag` depending of the type of each object currently seen as a generic `T` type from the parent class.)

Reply:
Your solution is great